src/Controller/EnMarche/InstitutionalEvents/DelegatedReferentInstitutionalEventsController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Controller\EnMarche\InstitutionalEvents;
+
+use App\Controller\AccessDelegatorTrait;
+use App\Controller\CanaryControllerTrait;
+use App\Entity\MyTeam\DelegatedAccess;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/espace-referent-partage/{delegated_access_uuid}/evenements-institutionnels", name="app_referent_institutional_events_delegated_", methods={"GET"})
+ *
+ * @Security("is_granted('ROLE_DELEGATED_REFERENT')")
```

Comment:
do we need to check `EVENT` access here?

Reply:
yes, fixed